include/swift/Driver/Util.h

```diff
@@ -45,6 +45,7 @@ namespace driver {
   struct FilelistInfo {
     enum class WhichFiles : unsigned {
       Input,
+      PrimaryInputs,
```

Comment:
This commit isn't "NFC", and probably shouldn't have gone in in this PR. Not that it's hurting anything, but right now it's unused code.